Yank/utils.py

```diff
@@ -1090,16 +1091,32 @@ def net_charge(self, value):
 
 # OpenEye functions
 # ------------------
-def is_openeye_installed():
+def is_openeye_installed(oetools=('oechem', 'oequacpac', 'oeiupac', 'oeomega')):
+    # Complete list of module: License check
+    tools_license = {'oechem':'OEChemIsLicensed',
+                     'oequacpac': 'OEQuacPacIsLicensed',
+                     'oeiupac': 'OEIUPACIsLicensed',
+                     'oeomega': 'OEOmegaIsLicensed'}
+    tool_keys = tools_license.keys()
+    # Cast oetools to tuple if its a single string
+    if type(oetools) is str:
+        oetools = (oetools,)
+    tool_set = set(oetools)
+    valid_tool_set = set(tool_keys)
+    if tool_set & valid_tool_set == set():
+        # Check for empty set intersection
+        raise ValueError
```

Comment:
Logic typo, My brain had that inside the `try..except` so I was trapping it in the same block.